docs/about/compatibility/openmp.md

```diff
@@ -280,6 +280,61 @@ to by “b” are in coarse-grain memory during and after the execution of the
 target region. This is accomplished in the OpenMP runtime library with calls to
 the ROCr runtime to set the pages pointed by “b” as coarse grain.
 
+#### Zero-copy behavior on MI300A and discrete GPUs
+
+The runtime behavior on MI300A and other discrete GPUs depends on the status of XNACK and the existence of the pragma `requires unified_shared_memory` as discussed in the following sections.
+
+##### Implicit zero-copy behavior on MI300A
```

Comment:
The introduction to this section got a bit garbled up due to successive updates and mixed up goals.
I separated "why it should work" from "user API to get what you need" and got the text below.
I would keep it to the second bit only (user API), instead of explaining why it works, which is the job of white papers, papers, openmp specifications, etc.
What do you think @SwRaw?


# why it works
Zero-copy is an implementation of OpenMP data mapping that does not result in GPU memory allocations and CPU-GPU memory copies, and
it is provided thanks to OpenMP's relaxed shared memory model. Mapping memory on a target region indicates that it will use the memory, and how it can be made consistent between CPU and GPU memory. While data mapping is optional on any GPU that provides native Unified Memory, it is convenient to obtain best performance when porting across MI300A and discrete memory GPUs. MI300A, MI300X, and MI200 are examples of AMDGPUs with [XNACK capability].


# how to get zero-copy
To obtain a zero-copy implementation on MI300A, users can either choose to add the [unified shared memory pragma](#unified-shared-memory-pragma) `requires unified_shared_memory`, which informs the compiler and runtime that the code must be compiled for and executed on GPUs providing Unified Memory through XNACK. Alternatively, programmers can choose to not use the requirement pragma but to rely on the OpenMP runtime to turn on zero-copy implementation. This happens when the OpenMP runtime detects it is running on an MI300A and the XNACK support is enabled in the environment in which the application is executed. This mode is called "implicit zero-copy".